test.py

```
@@ -0,0 +1,109 @@
+                next(iter(chess.SquareSet(chess.shift_up(chess.BB_SQUARES[sq]))))
+                for sq in adjacent_to_from_square if sq is not None
+            ]
+        elif from_rank == chess.RANK_NAMES.index('2'):
+            pawn_color = chess.BLACK
+            capturable_squares = [
+                next(iter(chess.SquareSet(chess.shift_down(chess.BB_SQUARES[sq]))))
+                for sq in adjacent_to_from_square
+                if sq is not None
+            ]
+        else:
+            assert False, 'Bad logic above'
```

Ruff (0.13.3)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)